### DISCUSSION TITLE
Books in folder sorted under 1 folder

### DISCUSSION THREAD

**Discussion by mokomonsen** [gotson/komga#1770] [Q&A] [answered]
I have alot of PDF's and EPUB files, and I set up Komga to sort them. But Komga keeps sorting files under a few categories. So books under \E-books\finance, are all sorted under finance, and not individual books. I tried to set up differerent libraries for each type, Finance, Bonsai, gardening, etc, but even then it just sort everything under one folder. How can I make Komga understand each is a different book and not a part of a series?

![image](https://github.com/user-attachments/assets/c0e64f73-c355-4919-97b7-31e26ef86e2f)

**Comment by gotson** [comment, marked as answer]
> How can I make Komga understand each is a different book and not a part of a series?

you can use the One-Shot feature: https://komga.org/docs/guides/oneshots

**Reply by mokomonsen** [reply]
Thank you! That did indeed solve it for me
